### DISCUSSION TITLE
Implement a canView() function for Pages

### DISCUSSION THREAD

**Discussion by andrewdwallo** [filamentphp/filament#5594] [Ideas]
Maybe a new function that implements both of these functions combined, for "Conditionally hiding pages in Navigation"

https://filamentphp.com/docs/2.x/admin/pages/getting-started#conditionally-hiding-pages-in-navigation

Could be useful when you just want to make a page available for a specific tenant or just a specific user id, name,condition,etc... not saying to remove the other functions or anything..

**Comment by danharrin** [comment]
The reason why this doesn't exist is because to be able to implement an authorisation check, we need to define a `mount()` method for Livewire, but if we do this then people can't add new parameters to their own mount() method via inheritance.

**Reply by andrewdwallo** [reply]
Understandable, not a big deal then.
